mail_outplacement_report/models/outplacement.py

```diff
@@ -25,18 +26,27 @@ def action_send_email(self):
 class Outplacement(models.Model):
     _inherit = "outplacement"
 
-    date = fields.Date(string="Meeting date", help="Date of meeting")
-    time = fields.Float(string="Meeting time", help="t.ex. 15:30")
-    performing_operation_adress = fields.Many2one(
+    performing_date = fields.Date(string="Meeting date", help="Date of meeting")
+    performing_time = fields.Float(string="Meeting time", digits=(12, 2), copy=False, help="t.ex. 15:30")
+    performing_operation_address = fields.Many2one(
         comodel_name='res.partner',
-        name='Adress', string='Performing Adress')
+        name='Adress', string='Meeting Address')
 
     @api.multi
     def action_send_eletter(self):
-        template_id = self.env.ref('mail_outplacement_report.email_template_assigned_coach').id
-        template = self.env['mail.template'].browse(template_id)
-        template.with_context(nadim_type='eletter').send_mail(
-            self.id, email_values={'notification': True}, force_send=True)
+        if not self.performing_operation_address:
+            raise ValidationError(_("Please enter meeting address before sending email"))
+        if not self.performing_date:
+            raise ValidationError(_("Please enter meeting date before sending email"))
+        elif self.performing_time == 0:
+            raise ValidationError(_("Please enter performing time before sending email"))
+        else:
+            template_id = self.env.ref('mail_outplacement_report.email_template_assigned_coach').id
+            _logger.warning("NADIM template_id: %s" % template_id)
+            template = self.env['mail.template'].browse(template_id)
```

Comment:
"self.env.ref('mail_outplacement_report.email_template_assigned_coach')" on line 44 will already return the same object as template on line 46. Remove this line and change line 44 to fetch the object instead of the id.